pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -39,12 +40,15 @@
 
 public class BrokerAdminApiApplication extends ResourceConfig {
   private static final String RESOURCE_PACKAGE = "org.apache.pinot.broker.api.resources";
+  public static final String PINOT_CONFIGURATION = "pinotConfiguration";
```

Comment:
Do you think it might be better to put this as a global constant to be shared among all instances?

Reply:
Yeah, I was thinking about the same, couldn't find an appropriate place. Have any suggestions?

Reply:
Let's keep it here for now. When we have more constants like this, we can clean them up then